pre_commit_hooks/make/phony-targets.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Make environment variables working in OSX GUI apps such as Github Desktop
+# https://stackoverflow.com/q/135688/483528
+export PATH=$PATH:/usr/local/bin
```

Comment:
this should a) not be needed as basic tools should not be installed there and b) should already be in the path as it is standard... 

```suggestion
```